Dockerfile

```diff
@@ -33,6 +32,12 @@ RUN cmake .
 RUN make 
 RUN make install
 
+WORKDIR /home/V2X-Hub/ext/date
```

Comment:
This needs to include instruction to clone the appropriate library. I was able to get it to build by adding the following
`WORKDIR /home/V2X-Hub/ext/
RUN git clone https://github.com/HowardHinnant/date.git
`

Reply:
This makes sense. thanks for the input. Changes  have been made to the dockerfile